tests/test_api.py

```diff
@@ -16,15 +16,8 @@
 
 
 class TestAPI(unittest.TestCase):
-    @classmethod
```

Comment:
no need to create this and it will break the timeout test if used since the paper broker expects a particular set of keys.